spec/defines/vhost_spec.rb

```diff
@@ -1197,7 +1197,7 @@
         }
       end
       let :facts do default_facts end
-      it { expect { is_expected.to compile }.to raise_error }
+      it { expect { is_expected.to compile }.to raise_error(RSpec::Expectations::ExpectationNotMetError) }
```

Comment:
I believe this should actually be ```it { is_expected.to raise_error(Puppet::Error) }```.

Reply:
Is there perhaps something wrong with my testing setup? I get this:
```
Failures:

  1) apache::vhost validation bad ensure should raise Puppet::Error
     Failure/Error: it { expect { is_expected.to compile }.to raise_error(Puppet::Error) }
     
       expected Puppet::Error, got #<RSpec::Expectations::ExpectationNotMetError: error during compilation: Evaluation Error: Error whil...h for Enum['absent', 'present'], got 'bogus' at line 3 on node xxx.domain.local> with backtrace:
         # ./spec/defines/vhost_spec.rb:1200:in `block (5 levels) in <top (required)>'
         # ./spec/defines/vhost_spec.rb:1200:in `block (4 levels) in <top (required)>'
     # ./spec/defines/vhost_spec.rb:1200:in `block (4 levels) in <top (required)>'
```

Reply:
You have to literally use my suggestion. The current syntax is a duplicate expectation which is not needed.

Reply:
Ahh, thanks. I made the change.